media/audio/audio_output_device_thread_callback.cc

```diff
@@ -60,7 +61,7 @@ void AudioOutputDeviceThreadCallback::Process(uint32_t control_signal) {
                      "callback_num", callback_num_, "frames skipped",
                      frames_skipped);
 
-  base::TimeDelta delay = base::Microseconds(buffer->params.delay_us);
+  base::TimeDelta delay = base::Microseconds(recordreplay::RecordReplayValue("audioparams::delay_us", buffer->params.delay_us));
```

Comment:
This looks good, it would be nice to have a comment on why we're recording/replaying this value though.  In general we need to explicitly record/replay values read from shared memory (even when the process doing the writing is another chromium process) because we don't have any facility to automatically capture those out-of-band writes when recording.  I ran into something similar in https://github.com/replayio/chromium/pull/508 a few days ago fwiw.